Tests/ConnectLibraryTests/ConnectTests/ConnectErrorTests.swift

```diff
@@ -87,17 +98,23 @@ final class ConnectErrorTests: XCTestCase {
 
     // MARK: - Private
 
-    private func errorData(expectedDetails: [ProtobufMessage]) throws -> Data {
+    private func errorData(expectedDetails: [ProtobufMessage], pad: Bool = true) throws -> Data {
         // Example error from https://connectrpc.com/docs/protocol/#error-end-stream
         let dictionary: [String: Any] = [
             "code": "unavailable",
             "message": "overloaded: back off and retry",
             "details": try expectedDetails.map { detail in
-                [
+                var val = try detail.serializedData().base64EncodedString()
```

Comment:
Swift best practices use full words for variable names ("value" and "errorDetail", for example)

Reply:
Ah gotcha, good to know. Thank you.